src/components/Button/Button.tsx

```diff
@@ -1,59 +1,131 @@
 import React, { FC } from "react";
+import styled, { css } from "styled-components";
+import { Color } from "../../theme/util";
 
 // https://www.w3schools.com/tags/tag_button.asp
-// https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
 export interface ButtonProps {
   ariaExpanded?: boolean | undefined;
   ariaPressed?: boolean | "mixed" | undefined;
+  ariaLabel?: string | undefined;
   autoFocus?: boolean;
   children: string | React.ReactNode;
   className?: string;
   disabled?: boolean;
   name?: string;
-  role: string;
   tabIndex?: string;
+  size?: "large" | "medium" | "small";
   type?: "button" | "reset" | "submit";
+  buttonVariety?: "text" | "textWithIcon" | "icon";
+  isSecondary?: boolean;
   value?: string;
   onClick?: () => void;
 }
 
 /**
  * Button
  */
+
+const BUTTON_HEIGHTS = {
+  large: 56,
+  medium: 50,
+  small: 44,
+};
+
+const StyledButton = styled.button<ButtonProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: ${(p) => (p.size ? BUTTON_HEIGHTS[p.size] + "px" : BUTTON_HEIGHTS.large + "px")};
```

Comment:
We have a linting issue where we are missing return type on the function. Either fix the linting issue or disable the rule in the esLint config.
Reference: https://github.com/typescript-eslint/typescript-eslint/blob/v2.27.0/packages/eslint-plugin/docs/rules/explicit-function-return-type.md.